### ISSUE TITLE
Settings title and description text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52479]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1.Launch VS Code.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown-->New settings editor.
3.Open Color Contrast Analyzer. 

**Actual**
Color contrast ratio is less than 4.5:1 for the content present on the page.i.e. Preview, Files,Editor etc.

**Expected:**
Color contrast ratio should be more than 4.5:1 for the content present on the page.i.e. Preview, Files,Editor etc.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The low visioned user will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment For Reference:**

![color](https://user-images.githubusercontent.com/39204909/41661027-51f63a90-74bb-11e8-809f-95b06100f73d.png)

![colorcontrast](https://user-images.githubusercontent.com/39204909/41661029-5227597c-74bb-11e8-8849-29dc179a4f28.png)

![colorcontrast_1](https://user-images.githubusercontent.com/39204909/41661031-52563f08-74bb-11e8-915b-b3b6a2451862.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
This only fails on our light theme

**Comment by misolori** [comment]
@ramya-rao-a which theme and what's the ratio you're getting? I'm seeing both of these passing the 4.5:1 ratio:

**Dark**
![image](https://user-images.githubusercontent.com/35271042/43560002-64b1ab94-95c5-11e8-97ae-74ceacbe1423.png)


**Light**
![image](https://user-images.githubusercontent.com/35271042/43559983-4c8c2bac-95c5-11e8-972f-77dcdc1742a8.png)

**Comment by ramya-rao-a** [comment]
Sorry, my bad

**Comment by AccessibilityTestingTeam-TCS** [comment]
The bug is verified on the below given build: 
version: 1.27.0-insider
The bug is fixed.
